bootstrap/src/lib/metrics/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as client from 'prom-client'
+
+client.collectDefaultMetrics()
+
+export const METRICS_ENABLED = !!process.env.METRICS
```

Comment:
From the Readme, should this be `process.env.METRICS_ENABLED`?

Reply:
Yep